.circleci/src/jobs/@stems-jobs.yml

```diff
@@ -0,0 +1,58 @@
+stems-build-storybook:
+  working_directory: ~/stems
+  docker:
+    - image: circleci/node:14.18
+  steps:
+    - checkout
+
+    # Download and cache dependencies
+    - restore_cache:
+        keys:
+          - dependency-cache-{{ checksum "packages/stems/package-lock.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - dependency-cache-
+
+    - run:
+        name: install dependencies
+        command: |
+          cd packages/stems
+          npm ci
+
+    - run:
+        name: build storybook
+        command: |
+          cd packages/stems
+          npm run build-storybook
+
+    - save_cache:
+        key: dependency-cache-{{ checksum "packages/stems/package-lock.json" }}
+        paths:
+          - ./packages/stems/node_modules
+
+    # Persist the build directory to a workspace
+    - persist_to_workspace:
+        root: ./
+        paths:
+          - packages/stems/storybook-static
+
+stems-deploy-storybook:
+  working_directory: ~/stems
+  docker:
+    - image: circleci/node:14.18
+  steps:
+    - checkout
+    - attach_workspace:
+        at: ./
+    - run:
+        name: install wrangler
+        command: "cd packages/stems && npm install @cloudflare/wrangler"
```

Comment:
You shouldn't need quotes here and you can even break it into two lines using the `|` operator!